assets/settings/default.json

```diff
@@ -335,6 +335,11 @@
     "include_ignored": false,
     "regex": false
   },
+  // These settings define whether the project search should search as you type, or only on enter.
+  "project_search": {
+    "automatic_submission": true,
```

Comment:
Nit: maybe you don't need to have a separate `automatic_submission`, as you could determine it based on whether the `automatic_submission_delay` is non-`null`.